.github/CODEOWNERS

```diff
@@ -75,8 +75,8 @@ src/platform/packages/shared/kbn-calculate-auto @elastic/obs-ux-management-team
 src/platform/packages/shared/kbn-calculate-width-from-char-count @elastic/kibana-visualizations
 x-pack/platform/plugins/private/canvas @elastic/kibana-presentation
 packages/kbn-capture-oas-snapshot-cli @elastic/kibana-core
-x-pack/platform/test/cases_api_integration/common/plugins/cases @elastic/kibana-cases
-src/platform/packages/shared/kbn-cases-components @elastic/kibana-cases
+x-pack/platform/test/cases_api_integration/common/plugins/cases @elastic/response-ops
```

Comment:
I wouldn't have expected to see CODEOWNER changes in this PR.

Reply:
me too, probably bad merge. I reverted it